lib/iot-core-mqtt-file-downloader/source/MQTTFileDownloader.c

```diff
@@ -411,76 +411,66 @@ static uint8_t handleJsonMessage( uint8_t * decodedData,
     return handleStatus;
 }
 
-bool mqttDownloader_handleIncomingMessage( MqttFileDownloaderContext_t * context,
-                                           MqttFileBlockHandler_t blockCallback,
-                                           char * topic,
-                                           size_t topicLength,
-                                           uint8_t * message,
-                                           size_t messageLength )
+bool mqttDownloader_isDataBlockReceived( MqttFileDownloaderContext_t * context,
+                                         char * topic,
+                                         size_t topicLength )
 {
     bool handled = false;
```

Comment:
Nit: `handled` seems like the wrong name now. Maybe something like `isDataBlock`?